### ISSUE TITLE
Colors in tables missing

### ISSUE THREAD

**Issue by ShaneBeee** [StylishThemes/Wikipedia-Dark#210]
I noticed the color is missing in tables. While this isn't a HUGE issue, it makes reading the table a little difficult.
How am I to keep track of my housewives...

* **Browser**: Safari (using Cascadea extension)
* **Operating System**: macOS 13.4
* **Link to page with the issue**: https://en.wikipedia.org/wiki/The_Real_Housewives_of_Beverly_Hills#Timeline_of_cast_members (don't judge me)
* **Screenshot**:
## Native Version with color in table:
<img width="1422" alt="Screenshot 2023-05-31 at 8 44 51 PM" src="https://github.com/StylishThemes/Wikipedia-Dark/assets/20008482/3d65a89d-6e2e-4ca0-a9e5-87d6cfd9f742">

## Wiki-Dark without color in table
<img width="1422" alt="Screenshot 2023-05-31 at 8 44 44 PM" src="https://github.com/StylishThemes/Wikipedia-Dark/assets/20008482/ae4e84cf-a362-4f5f-a23f-ef352665ec73">

**Comment by AfroThundr3007730** [comment]
Yeah, with the current way we used the `!important` hammer to strip a lot of the custom wikitable colors, there's many places where it affects color coding like this. We've been selectively adding rules to override those colors with ones from our palette, but we've only been targeting the more standardized colors in well known templates.

Fortunately, this is one of those cases that could be fixed easily with the latter approach. I'll take a look at fixing this later, or if you feel like getting your hands dirty, pull requests are welcome.

**Comment by pointydev** [comment]
In my opinion, it'd be nice to have a UserCSS option to completely disable colour overrides in wikitable cells that have custom colours. Some tables use colour legends to convey information, and I would much rather be blinded slightly and consume that information that have to turn off the entire theme and get flashbanged. An example of this can be found [here](https://en.wikipedia.org/wiki/LK-99#Experimental_studies):

![dark](https://github.com/StylishThemes/Wikipedia-Dark/assets/22241432/1b6745cb-6b49-4460-bc6b-45040156f845)
![light](https://github.com/StylishThemes/Wikipedia-Dark/assets/22241432/6e8c51bc-40cf-4089-9bab-0b66fa0ce51a)

Thanks,
Elliott

**Comment by xeophyte** [comment]
Same here 
![image](https://github.com/StylishThemes/Wikipedia-Dark/assets/121556803/431e55b9-869d-42f1-9152-5e21d444fc7d)

**Comment by neocharles** [comment]
I also noticed the coloring in the tables on https://en.wikipedia.org/wiki/Multicast_address has been removed.

**Comment by AfroThundr3007730** [comment]
As is, these colors are made with inline styles in most places as the practice of using classes and template styles didn't happen until fairly recently in Wikipedia editing history. Hence, the reason they're so prevalent, and such pervasive use of `!important` overrides everywhere to capture them all. It's gotten to be messy, and cleaning it up is easier said than done.

We concluded a while back that the best way to solve it would be a rewrite of the non-mobile style (we already rewrote the mobile one), but since then, the Vector Modern skin was released and made default, and it's grown it's own dark mode, reducing the priority of a rewrite.

If anyone is interested in finding all the places that happen to affect these tables (there's a lot) and putting them behind an option, pull requests are certainly welcome.

**Comment by MMachado05** [comment]
On the modern design, there's also a glitch where the current section becomes unreadable:
![image](https://github.com/user-attachments/assets/15a44341-cd14-4fe5-a671-f20752bea94e)
